config.json

```diff
@@ -66,6 +66,7 @@
     "wordy",
     "allergies",
     "poker",
+    "all-your-base",
```

Comment:
Ok, Thanks. That document is out of date already. (Which is not at all your fault! Thanks for following instructions.)